sympy/functions/special/tests/test_delta_functions.py

```diff
@@ -39,11 +39,11 @@ def test_DiracDelta():
     assert DiracDelta(sqrt(x)).is_simple(x) is False
     assert DiracDelta(x).is_simple(y) is False
 
-    assert DiracDelta(x*y).simplify(x) == DiracDelta(x)/abs(y)
-    assert DiracDelta(x*y).simplify(y) == DiracDelta(y)/abs(x)
-    assert DiracDelta(x**2*y).simplify(x) == DiracDelta(x**2*y)
-    assert DiracDelta(y).simplify(x) == DiracDelta(y)
-    assert DiracDelta((x - 1)*(x - 2)*(x - 3)).simplify(x) == \
```

Comment:
Just leaving them in place is not working fine. This makes the tests fail. 
We need to put all off them under `with raises(SymPyDeprecationWarning):` . This made them pass the tests. And I did that.

I am not sure because none of the deprecated methods have there tests in sympy. So I didn't get a good example to follow.